dlp/snippets/inspect/inspect_test.go

```diff
@@ -276,3 +276,25 @@ func TestInspectBigquery(t *testing.T) {
 		})
 	}
 }
+
+func TestInspectStringWithExclusionDictSubstring(t *testing.T) {
+	tc := testutil.SystemTest(t)
+	buf := new(bytes.Buffer)
+
+	if err := inspectStringWithExclusionDictSubstring(buf, tc.ProjectID, "Some email addresses: gary@example.com, TEST@example.com", []string{"TEST"}); err != nil {
+		t.Errorf("inspectStringWithExclusionDictSubstring: %v", err)
```

Comment:
issue: use `t.Fatal(err)` here instead of `t.Errorf()`. If this check fails, the other asserts will fail too.

Reply:
fixed it.